src/lib/enabled-integration-products.ts

```diff
@@ -0,0 +1,7 @@
+import { ProductSlug } from 'types/products'
+
+// The Products that have Integrations enabled
+export const ENABLED_INTEGRATION_PRODUCTS: Array<ProductSlug> = [
```

Comment:
💭 We could store this in the[ config data](https://github.com/hashicorp/dev-portal/blob/main/config/base.json) for dev & preview, it's made [accessible via `__config.<property>`](https://github.com/hashicorp/dev-portal/blob/main/next.config.js#L85)

Reply:
Oh neat I really like this.

I'll adjust this in the near future to use this strategy; I think I need to think some more about how I'd structure that / what our setups will be so I'll still merge this PR in here, but 100% will switch to this config strategy.